attributes/network/1000_abci_backend.adoc

```diff
@@ -16,5 +16,52 @@ It exposes the following endpoints:
 - `abci.endBlock`.
 - `abci.init`.
 - `abci.info`.
+- `abci.checkTx`.
```

Comment:
Couple of comments;

1. this seems to be related to the frontend (#1001), not the backend (#100).
2. these attributes are better related to tendermint than ABCI, and as such should probably go into `tendermint` namespace (e.g. `tendermint.checkTx`).